examples/polyglot-zipkin/java-wildfly-swarm/README.adoc

```diff
@@ -0,0 +1,17 @@
+= Example application of Wildfly Swarm with brave
+
+Wildfly Swarm application instrumented with zipkin brave library.
+
+== Run & Build
+[source,shell]
```

Comment:
Same as comment above - not sure if all services can be run standalone, so may need to distinguish. If all can be run in either docker or standalone, then I may have missed that ,